packages/commonwealth/client/scripts/views/components/sidebar/SidebarNotificationIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import '@knocklabs/react-notification-feed/dist/index.css';
+import React from 'react';
+import './SidebarNotificationIcon.scss';
+
+type SideBarNotificationIconProps = {
+  unreadCount: number;
+};
+
+export const SideBarNotificationIcon = ({
+  unreadCount,
+}: SideBarNotificationIconProps) => (
+  <div className="sidebar-notification-icon">
```

Comment:
```suggestion
  <div className="SideBarNotificationIcon">
```

hey, I mentioned it multiple times, even in this PR